### ISSUE TITLE
Cannot add mp3 file when creating a project

### ISSUE THREAD

**Issue by rekkitcwts** [gyunaev/karlyriceditor#16]
Tried to start a new project but it will not add the mp3 file. Instead it gives errors like "Cannot open the music file. Your audio device cannot play this format." Already tried renaming the file to a shorter one, converting it to other formats like ogg, but it's still the same error. The terminal did not even show any error messages even though this is compiled from source and the program is being run from the terminal.
Even opening the "I Will Survive" example gave an error with a slightly different message, my home directory is /home/pi but the directory mentioned in the error is /home/tim.
EDIT: After opening the "I Will Survive" example, the project settings gave an option to change the file name of the music file but it also gave the same error as when creating a new project -- also "cannot play the music file, your audio device cannot play this format"

**Comment by Symbian9** [comment]
> "Cannot open the music file. Your audio device cannot play this format."

Same issue.

- *karlyriceditor* version: 3.2
- Platform/OS: Debian 11
- DEB-package: https://www.deb-multimedia.org/dists/stretch/main/binary-amd64/package/karlyriceditor

**Comment by brenodantas10** [comment]
Same issue...

- _karlyriceeditor_ version: 3.3
- Platform/OS: PopOS 22.04 (ffmpeg 4.4.2 + Qt6)
- Compiled from this repo with tag 3.3

**Comment by gyunaev** [comment]
Try with 4.0 please. If it doesn't work, please make sure the file can be played with `gst-play` cmdline tool. If it cannot play with the tool, your codecs/ffmpeg is incomplete (some distros ship parts of ffmpeg only, for example).

**Comment by brenodantas10** [comment]
Yes, now with 4.0 it works perfectly for creating a project and play the audio. Could export normaly to CD+G as well with LRC v2 (I've not tested other types).

- Platform/OS: PopOS 22.04
- Compiled with Qt 6.9 and tag 4.0 -> Ubuntu 22.04 repo does not have a Qt that met the requirements for this project, had to download from Qt.io
- Had to get libgstreamer1.0-dev and libgstreamerd-3-dev (dunnot why this last one, but it couldn't find gstreamer-app development packages without it)

**Comment by gyunaev** [comment]
Perfect! Please also try the video export.
